### PR TITLE
adds 4 get articleById

### PR DESCRIPTION
**Pull request by smpezert**
I finished with 4 get article by ID. Is it anything that is missing?

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -42,3 +50,26 @@ describe("3. GET /api/topics", () => {
...
+
+describe("4. GET /api/articles/:article_id", () => {
+  test("status 200: responds with an article object containing the following properties", () => {
+    const article_id = 1;
+    return request(app)
+      .get(`/api/articles/${article_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body).toEqual(
```
Comment (Poonam-raj) [inline __tests__/app.test.js:51]: I'd like you to specifically check for the right articleID to come back in this test (aka 1)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -21,6 +21,14 @@ describe("Sad Paths", () => {
         expect(msg).toBe("Invalid path");
       });
   });
+  test("status 400: responds for bad requests", () => {
```
Comment (Poonam-raj) [inline __tests__/app.test.js]: This sad path test should be under the describe block on line 54, keep your sad path testing with the endpoint it relates to - not separate blocks

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Just some minor changes to go but it's looking very strong 🎉

File: __tests__/app.test.js (hunk 1 above)
Comment (smpezert) [reply __tests__/app.test.js:51]: Added one more test with a specific article!

File: __tests__/app.test.js (hunk 2 above)
Comment (smpezert) [reply __tests__/app.test.js]: Added more tests and transferred to the right places

**Pull request merged**